Auth/app/src/main/java/com/example/Auth/ChangeInfoActivity.kt

```diff
@@ -0,0 +1,133 @@
+package com.example.Auth
+
+import android.app.Activity
+import android.content.Intent
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.net.Uri
+import android.os.Build
+import android.os.Bundle
+import android.provider.MediaStore
+import android.util.Log
+import android.widget.*
+import androidx.annotation.RequiresApi
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.serialization.*
+import kotlinx.serialization.json.*
+import java.io.ByteArrayOutputStream
+import java.lang.Short.decode
+import java.nio.charset.StandardCharsets
+import java.util.Base64
+
+
+
+class ChangeInfoActivity : AppCompatActivity() {
+
+    class ChangeInfoActivityUnits {
+        var applyButton: Button? = null
+        var cancelButton: Button? = null
+        var editTextName: TextView? = null
+        var editTextBio: TextView? = null
+        var imageViewProfilePic: ImageView? = null
+        var profilePicBitMap: Bitmap? = null // Profile picture bitmap
+    }
+
+    private var units: ChangeInfoActivityUnits = ChangeInfoActivityUnits()
+
+    @RequiresApi(Build.VERSION_CODES.O)
+    fun bitMapToBase64String(bitMap: Bitmap?): String? {
```

Comment:
If same code is used in multiple files, it should be moved to a single place. Copying it everywhere is a bad practice